python/test/Ice/exceptions/ServerAMD.py

```diff
@@ -178,7 +29,7 @@ def run(self, args: list[str]):
             adapter = communicator.createObjectAdapter("TestAdapter")
             adapter2 = communicator.createObjectAdapter("TestAdapter2")
             adapter3 = communicator.createObjectAdapter("TestAdapter3")
-            object = ThrowerI()
+            object = TestI.ThrowerI()
```

Comment:
We were testing both AMD and non AMD, how does this work now?

Reply:
I got mixed up when I updated this and thought that they had the same impl. Will fix.